### ISSUE TITLE
As CLI tool?

### ISSUE THREAD

**Issue by mikecann** [playcanvas/playcanvas-sync#30]
Any reason why this isnt simply a CLI tool on npm? Also why the explicit calling of node? This is kind of counter-intuitive to people coming from the web ecosystem. 

A more usual / preferred developer experience would be:

1. `npm install playcanvas-sync -g`
2. `playcanvas-sync -c my-config.json watchAll`

**Comment by willeastcott** [comment]
I agree with this 100%. `playcanvas-sync` should definitely be distributed and work as you describe IMHO.

**Comment by zachpeterpaul** [comment]
After 'npm install' from the project folder you can run it without 'node' or '.js'. The example command in the readme is for less experienced users who may not want to install it. './pcwatch.js' would also work.

**Comment by willeastcott** [comment]
@zachpeterpaul Not on Windows, it doesn't.

**Comment by willeastcott** [comment]
I've just published version [2.0.0](https://github.com/playcanvas/playcanvas-sync/releases/tag/v2.0.0) to NPM. Now you can simply do:

```bash
npm install -g playcanvas-sync
```

And then run `pcsync` or `pcwatch` from the CLI.

Closing.
